roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
```

Comment:
sorry but I am still -1 for this.

this is more of organization issue and breaks consistency with rest of the roles organization. right now `contiv_network` is the role for configuring contiv networking which shall include both aci and standalone deployments.